build_projects/dotnet-cli-build/MsiTargets.cs

```diff
@@ -258,5 +172,11 @@ private static void ReattachEngineToBundleHelper(string bundle, string engine)
 
             File.Delete(engine);
         }
+
+        private static string GetEngineName(string bundle)
```

Comment:
No, it happened during refactoring of this file. It doesn't affect anything though.